packages/node/src/integrations/http.ts

```diff
@@ -90,7 +103,26 @@ type WrappedRequestMethodFactory = (original: OriginalRequestMethod) => WrappedR
 function _createWrappedRequestMethodFactory(
   breadcrumbsEnabled: boolean,
   tracingEnabled: boolean,
+  tracePropagationTargets: (string | RegExp)[] | undefined,
 ): WrappedRequestMethodFactory {
+  // We're caching results so we dont have to recompute regexp everytime we create a request.
+  const urlMap: Record<string, boolean> = {};
+  const shouldAttachTraceData = (url: string): boolean => {
+    if (tracePropagationTargets === undefined) {
+      return true;
+    }
```

Comment:
This is totally optional for now; leaving this more as a note:  I think that handling the default case like this is fine for the Node SDK for now but in browser, we'll have a set of default values we have to check the incoming URL against. So whenever we add this option to the browser SDK we, might want to extract this function and have a combined default value handling logic. 
So at that time, we might want to come back and define `/.*/` as the default value for Node and match strings against it.

Reply:
Good point - I agree, let's keep this in mind when we do the browser changes!